### PR TITLE
Generate a random board upon start

### PR DESCRIPTION
**Pull request by mjordonez2001**
### What?
- Game is able to generate a random board.
- Game generates a random solvable board on start.

### CODE REVIEW

File: src/tile-board/Tileboard.jsx [hunk 1]
```diff
@@ -78,4 +78,29 @@ function toggle(arrayGrid, y, x, size) {
...
+
+  for (let i = 0; i < size * size; i++) {
+    const x = randomTilePosition(size);
+    const y = randomTilePosition(size);
+
+    tempArray = toggle(tempArray, y, x, size);
+  }
+
+  return tempArray;
+}
```
Comment (nick-telsan) [inline src/tile-board/Tileboard.jsx]: Personal preference: I like to keep these kind of functions in a separate `utils.js` or `functions.js` file and let my jsx files focus primarily on rendering elements and doing React stuff.

**Review by nick-telsan — APPROVED** [review]
This makes sense to me.

Food for future thoughts: Not every board is solvable. How could you generate a random board that is always solvable?

**Comment by mjordonez2001** [comment]
> This makes sense to me.
> 
> Food for future thoughts: Not every board is solvable. How could you generate a random board that is always solvable?

If I'm not mistaken, going from all lights on to all lights off is always possible for any n x n (square) board. So as long as I start with an all lights on board, and randomly toggle tiles to generate a random board, I should be fine. ([Source](https://mathworld.wolfram.com/LightsOutPuzzle.html#:~:text=As%20shown%20by%20Sutner%20(1989)%2C%20going%20from%20all%20lights%20on%20to%20all%20lights%20off%20is%20always%20possible%20for%20any%20size%20square%20lattice))

File: src/tile-board/Tileboard.jsx (hunk 1 above)
Comment (mjordonez2001) [reply src/tile-board/Tileboard.jsx]: Totally makes sense

**Comment by nick-telsan** [comment]
> > This makes sense to me.
> > Food for future thoughts: Not every board is solvable. How could you generate a random board that is always solvable?
> 
> If I'm not mistaken, going from all lights on to all lights off is always possible for any n x n (square) board. So as long as I start with an all lights on board, and randomly toggle tiles to generate a random board, I should be fine. ([Source](https://mathworld.wolfram.com/LightsOutPuzzle.html#:~:text=As%20shown%20by%20Sutner%20(1989)%2C%20going%20from%20all%20lights%20on%20to%20all%20lights%20off%20is%20always%20possible%20for%20any%20size%20square%20lattice))

Yep! That is exactly right!

**Review by solomonhawk — APPROVED** [review]
LGTM 👍

**Pull request merged**